### ISSUE TITLE
[Pytanie] Zapytanie o status blokady sprzedaży

### ISSUE THREAD

**Issue by Uriziel01** [allegro/allegro-api#766]
Witam,
czy istnieje zasób (w starym lub nowym API) którym można jakkolwiek odpytać o status blokady konta?
Chodzi mi o blokadę nałożoną na wystawianie na sprzedaż z komunikatem:
```
Nie możesz wystawiać przedmiotów na sprzedaż.
Powód ograniczenia otrzymałeś e-mailem. Napisz do nas, aby to wyjaśnić.
```

**Comment by PawelTaberski** [comment]
Nie ma takiego zasobu w API zgodnie z komunikatem musisz w takim wypadku w celu wyjaśnienia skontaktować się mailowo.

**Comment by PawelTaberski** [comment]
Tak, jak pisałem nie ma do tego dedykowanego zasobu. Jednakże jeśli będzie chciał wystawić ofertę dostaniesz odpowiedni komunikat.
